pdsfile/pdsfile_test_helper.py

```diff
@@ -39,22 +40,41 @@ def read_or_update_golden_copy(data, path, update):
     """
 
     path = Path(path)
-    # Create the golden copy by using the current output
+    # Create the golden copy by using the current output if or the update param is given
+    # or the golden copy doesn't exist.
     if update or not path.exists():
         # create the directory to store the golden copy if it doesn't exist.
         os.makedirs(os.path.dirname(path), exist_ok=True)
-
-        # write the associated_abspaths output to the file.
-        with open(path, 'w') as f:
-            pprint.pp(data, stream=f)
-        print('\nCreate the golden copy', path)
+        # write the output to the file.
+        write_data_to_file(data, path)
         return 0
 
+    return read_file(path)
+
+def read_file(path):
+    """Return data from the read file
```

Comment:
BTW if we're going to start putting docstrings in, we should make them in Google Docstyle format. Less work to do later to change them.